AzureMonitorAgent/ama_tst/modules/install/check_ama.py

```diff
@@ -1,30 +1,41 @@
 import re
-import requests
 import xml.dom.minidom
+import urllib
+try:
+    import requests
+except ImportError:
+    pass
 
 from error_codes import *
 from errors      import error_info, get_input
 from helpers     import get_package_version
+from connect.check_endpts import check_internet_connect
 
 AMA_URL = 'https://docs.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-extension-versions'
 
 def get_latest_ama_version(curr_version):
```

Comment:
If possible, can we leave a comment (i believe you can create an 'invisible' comment in markdown with `[//]: # some comment`) that the [table here](https://github.com/MicrosoftDocs/azure-docs-pr/blob/main/articles/azure-monitor/agents/azure-monitor-agent-extension-versions.md#version-details) should not be changed without consulting glinuxagent@microsoft.com team, since it will break AMA TST?

Reply:
Taking care of it for speed purposes https://github.com/MicrosoftDocs/azure-docs-pr/pull/218340